### ISSUE TITLE
NodeJS required version 

### ISSUE THREAD

**Issue by traversaro** [cciliber/YarpJS#4]
The README specifies that is necessary to have a version of NodeJS `>= 4.12`. However there is no 4.12 NodeJS version, at least on the official website: https://nodejs.org/en/download/releases/ . (The latest releases are 4.4.4 and 6.2.0 ). 

I am a NodeJS newbie, so I am probably missing something related to this.

**Comment by cciliber** [comment]
Hi @traversaro, sorry for the belated reply, for some reason I am still not getting notified when someone crease a new issue. 

Anyways: I fixed the docs (version >= 4.2.2 is fine). I messed up with Node.js versioning (they have in parallel version 6.x, 5.x and 0.12.x) see [this](http://stackoverflow.com/questions/32707088/nodejs-version-number-confusion) stack overflow post for more details.
